src/api/tests/service/user/test_user_identify.py

```
@@ -46,6 +46,78 @@ def _reset_shifu_tables() -> None:
+
+    from flaskr import dao
+    import flaskr.service.user.phone_flow as phone_flow
+    from flaskr.service.user.consts import (
+        USER_STATE_REGISTERED,
+        USER_STATE_UNREGISTERED,
+    )
+    from flaskr.service.user.models import AuthCredential, UserInfo as UserEntity
+
+    app = Flask(__name__)
+    app.config.update(
+        SECRET_KEY="test-secret-key",
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to argument: "SECRET_KEY"

(S106)